docs/iris/src/whatsnew/2.2.rst

```diff
@@ -25,11 +25,78 @@ Iris 2.2.0 Features
   contiguous, or the cube's data must be masked at the discontiguities in
   order to avoid plotting errors.
 
-  The iris plot functions :func:`iris.plot.quiver` has been added, and this
+  The iris plot function :func:`iris.plot.quiver` has been added, and this
   also works with 2-dimensional plot coordinates.
 
 .. admonition:: 2-Dimensional Grid Vectors
 
   The iris functions :func:`iris.analysis.cartography.gridcell_angles` and
   :func:`iris.analysis.cartography.rotate_grid_vectors` have been added,
   allowing you to convert gridcell-oriented vectors to true-North/East ones.
+
+* The :class:`iris.fileformats.um.FieldCollation` objects, which are passed
+  into load callbacks when using
+  :func:`iris.fileformats.um.structured_um_loading`, now
+  have the additional properties:
+  :data:`iris.fileformats.um.FieldCollation.data_filepath` and
+  :data:`iris.fileformats.um.FieldCollation.data_field_indices`.
+  These provide the file locations of the original data fields, which are
+  otherwise lost in the structured loading process.
+
+* NetCDF data variable chunk sizes are utilised at load time for significant
+  performance improvements.
+
+* :func:`iris.util.reverse` can now be used to reverse a cube by specifying
+  one or more coordinates.
+
+* Time mean fields can now be saved to PP files as a cell method.
+
+* Cube aggregators :func:`iris.analysis.MIN`, :func:`iris.analysis.MAX`,
+  :func:`iris.analysis.SUM` and :func:`iris.analysis.COUNT` now perform lazy
+  aggregation by utilizing Dask.
```

Comment:
Meanwhile, dask _isn't_ a proper noun, so could you remove the capitalization here please 😜